packages/frontend/editor-ui/src/views/NodeView.vue

```diff
@@ -1186,6 +1175,20 @@ function onClickConnectionAdd(connection: Connection) {
 	});
 }
 
+function onClickReplaceNode(nodeId: string) {
+	const node = workflowsStore.getNodeById(nodeId);
+	if (!node) return;
+	const nodeType = nodeTypesStore.getNodeType(node.type);
+	if (!nodeType) return;
+
+	nodeCreatorReplaceTargetId.value = nodeId;
+	if (isTriggerNode(nodeType)) {
+		nodeCreatorStore.openNodeCreatorForTriggerNodes(NODE_CREATOR_OPEN_SOURCES.REPLACE_NODE_ACTION);
+	} else {
```

Comment:
This doesn't seem to cover AI nodes, so it defaults to regular mode when trying to replace agent sub-node (like model or tool).